OcarinaSongs.py

```diff
@@ -424,11 +439,17 @@ def generate_song_list(world: World, frog: bool, warp: bool) -> dict[str, Song]:
         raise ShuffleError('Could not generate random songs')
 
     # sort the songs by length
-    random_songs.sort(key=lambda s: s.difficulty)
+
```

Comment:
I don't think this can just be removed? Now the randomly generated songs won't be assigned based on the difficulty heuristic anymore.

Reply:
Does sorting by difficulty really make sense for the player?
Personally i had to dig in the code to see what exactly was the sorting method.
I think i'd make more sense to just display them all in the same order regarding of the settings.

Reply:
Unless I'm misunderstanding your changes to the code, this sorting determines which melody is assigned to which song. For example, ZL will usually be an easy melody, while Nocturne will usually be a difficult one.

Reply:
Ah i thought this would only impact how they are shown in the spoiler log. I'll check tonight if it's more than that.

Reply:
Ok yeah you were right i don't know why i removed this sort, we can just sort it again right after for spoiler log and this display function.